tools/samtools_sort.cwl

```diff
@@ -8,34 +8,34 @@ requirements:
     dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/samtools:1.9'
   - class: InlineJavascriptRequirement
   - class: ResourceRequirement
-    coresMin: 16
-    ramMin: 24000
+    coresMin: $(inputs.cores)
+    ramMin: ${ return inputs.cores * 1000 }
```

Comment:
1 GB per core? Why did it come down from 24 GB?

Reply:
Because of this arg: `-m 1G`
24000 was kind of random I think

Reply:
Sounds fine to me.